build/release.mk

```diff
@@ -31,11 +31,23 @@ ifeq ($(GENERATE_PACKAGES), true)
 	# Pre-release/release actually builds and uploads images
 	# goreleaser will compile binaries, generate manifests, and push multi-arch docker images
 	# TAG_SUFFIX should be set as "-pre" during prereleases
-	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser.yml --skip-validate --rm-dist
+	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser.yml --skip=validate --clean
 else
 	@echo "===> $(INTEGRATION) === [release/build] build compiling all binaries"
 	# release/build with PRERELEASE unset is actually called only from push/pr pipeline to check everything builds correctly
-	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip-validate --snapshot --rm-dist
+	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip=validate --snapshot --clean
+endif
+
+.PHONY : release/build-fips
+release/build-fips: release/deps release/clean
+ifeq ($(GENERATE_PACKAGES), true)
+	@echo "===> $(INTEGRATION) === [release/build] PRERELEASE/RELEASE compiling fips binaries, creating packages, archives"
+	# TAG_SUFFIX should be set as "-pre" during prereleases
+	@$(GORELEASER_BIN) release --config $(CURDIR)/.goreleaser-fips.yml --skip=validate --clean
+else
+	@echo "===> $(INTEGRATION) === [release/build-fips] build compiling fips binaries"
+	# release/build with PRERELEASE unset is actually called only from push/pr pipeline to check everything builds correctly
```

Comment:
I think this comment is wrong or in the wrong place.
The if/else is checking GENERATE_PACKAGES, and we are not calling it set to false